lstchain/scripts/onsite/paths.py

```diff
@@ -0,0 +1,9 @@
+import os
```

Comment:
I think this should not go into a `scripts` subfolder, maybe just `lstchain.onsite`?

Reply:
ok, I moved the file and put it where you initially suggested